spec/features/companies/companies_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Company', type: :feature do
+  let(:admin) { create(:user, :super_admin) }
+  let!(:company) { create(:company, :uk) }
+  let!(:news_list) { create_list(:article, 9, :published, :uk) }
+  let!(:docs_list) { create_list(:attachment, 12) }
+
+  before do
+    visit companies_path(locale: I18n.locale)
+  end
+
+  context 'when user visits company page' do
+    it 'index page - company part' do
+      expect(page).to have_current_path companies_path(locale: I18n.locale)
+      expect(page).to have_content(Date.today.year - company.start_year)
+      expect(page).to have_content(company.projects_count)
+      expect(page).to have_content(company.clients_count)
+      expect(page).to have_content(company.grants_count)
+      expect(page).to have_content(company.text_about)
+    end
+
+    it 'index page - news part' do
+      expect(page).to have_current_path companies_path(locale: I18n.locale)
+      news_list.each do |news|
+        expect(page).to have_content news.title
```

Comment:
use one style 

`expect(page).to have_content(news.title)`

Reply:
put this under next check - it would be pettier

Reply:
updated